x-pack/plugins/upgrade_assistant/public/application/components/overview/upgrade_step/upgrade_step.tsx

```diff
@@ -7,119 +7,33 @@
 
 import React from 'react';
 
-import {
-  EuiText,
-  EuiFlexItem,
-  EuiFlexGroup,
-  EuiSpacer,
-  EuiButton,
-  EuiButtonEmpty,
-} from '@elastic/eui';
+import { EuiText, EuiSpacer } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import type { EuiStepProps } from '@elastic/eui/src/components/steps/step';
-import { useAppContext } from '../../../app_context';
 
 const i18nTexts = {
   upgradeStepTitle: i18n.translate('xpack.upgradeAssistant.overview.upgradeStepTitle', {
-    defaultMessage: 'Upgrade to Elastic 8.x',
+    defaultMessage: 'Get ready for the next version of Elastic',
```

Comment:
This steps seems odd to me. Isn't the entire page + other steps about getting ready for the upgrade?

I'd advise the user to check back at this page when it's time to upgrade to 8.x.

```suggestion
    defaultMessage: 'Check back here when it's time to upgrade to 8.0',
```

Reply:
@sabarasaba What do you think about implementing this new step title or something similar?